plasmapy/diagnostics/langmuir.py

```diff
@@ -67,39 +69,40 @@ def langmuir_analysis(potential_sweep, measured_current, probe, ion='H',
     -------
     T_e : Quantity
         Best estimate of the electron temperature in units of eV.
-        
+
     n_e : Quantity
         Best estimate of the electron density in units of m^-3.
-        
+
     n_i : Quantity
         Best estimate of the ion density in units of m^-3. Equal to n_e
         unless the OML method is used.
```

Comment:
This could be replaced with the average ionization. You then set a default ionization of 1 so that n_e=n_i for most cases.

Reply:
Well, there might be a large difference in errors of the obtained parameters. For example, the ion density obtained with OML theory is often significantly more accurate than the electron density obtained from the electron saturation current. So cases exist where there is a large difference between `n_i` and `n_e`, but only because one method is simply better than the other. If then a strange ionization value is returned this might sent the wrong message to users.

Reply:
Ah, yes I forgot that each of these are just initial values that are supposed to converge. Then this is fine, just make sure note this point and that the n_e and n_i that you get won't necessarily give you an accurate average ionization.